test/SlowTests/Client/TimeSeries/BulkInsert/BulkInsert.cs

```diff
@@ -1124,30 +1124,45 @@ public void CanAppendALotOfValuesAndTimeSeriesInParallel(int numberOfValues)
                     session.SaveChanges();
                 }
                 var count = new CountdownEvent(numberOfTimeSeries);
+
+                var debugData = new string[numberOfTimeSeries];
+
                 Parallel.For(0, numberOfTimeSeries, async (i) =>
```

Comment:
Pretty sure that this is wrong, the `async` on the `Paralle.For`, who waits for this to complete?

Reply:
It bugs me that we are doing it in this manner. It _works_, but it isn't a good way to handle this.
Bad pattern for multi thread ing / syncronizations

Reply:
Probably we can switch to `Parallel.ForEachAsync(Enumerable.Range(0, numberOfTimeSeries), (i, token) => ...)`

Reply:
done